analyses/molecular-subtyping-LGAT/01-make-lgat-final-table.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "LGAT final table for BRAFV600E and BRAF fusion"
+output: html_notebook
+---
+
+This script uses subsetted LGAT wgs file and fusion files from data release to identify samples with BRAF V600E mutations and BRAF fusion
+
+```{r}
+library("tidyverse")
+```
+
+
+### Read files
+    
+```{r}
+
+# to get root directory
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+# get clinical file for participant and sample_id columns associated with bs_ids
+clinical <- readr::read_tsv(file.path(root_dir,"data", "pbta-histologies.tsv")) %>%
+  select("Kids_First_Biospecimen_ID","Kids_First_Participant_ID","sample_id","short_histology")
+# get subsetted wgs file
+subsetWgs<-readr::read_tsv(file.path(root_dir,"analyses","molecular-subtyping-LGAT","lgat-subset","LGAT_snv_subset.tsv")) %>% 
+  dplyr::select("Tumor_Sample_Barcode","BRAF_V600E")
+# get BRAF fusion list per sample
+brafFusion<-readr::read_tsv(file.path(root_dir, "data","pbta-fusion-recurrently-fused-genes-bysample.tsv")) %>%
+  dplyr::select(Sample,BRAF) %>%
+  dplyr::filter(BRAF==1)
+# get putative oncogene fusion list
+kiaabrafFusion<-readr::read_tsv(file.path(root_dir, "data","pbta-fusion-putative-oncogenic.tsv")) %>%
+  dplyr::select(Sample, FusionName) %>% unique() %>%
+  dplyr::filter(grepl("KIAA1549--BRAF",FusionName))
```

Comment:
It doesn't look to me like we ever save a file that notes the presence of this _specific_ fusion (rather than any _BRAF_ fusion), although I know this specific fusion is of interest. What is your rationale for including this step rather than just going by what is in `pbta-fusion-recurrently-fused-genes-bysample.tsv`?

Reply:
I was wondering if different braf fusions are present in LGAT but it seems to be all KIAA1549-BRAF so I can just using pbta-fusion-recurrently-fused-genes-bysample.tsv here

Reply:
That makes sense, thanks!